auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java

```diff
@@ -679,6 +684,70 @@ public AuthenticationConfiguration usePassword(String password) {
         return usePassword(password == null ? null : ClearPassword.createRaw(ClearPassword.ALGORITHM_CLEAR, password.toCharArray()));
     }
 
+    /**
+     * Create a new configuration which is the same as this configuration, but converts the given masked password to a
+     * clear password and uses the clear password to authenticate.
+     *
+     * @param password the password to use
+     * @return the new configuration
```

Comment:
Would be good to add an ```@throws``` line here as well.

Reply:
Fixed